src/tests/profiler/native/releaseonshutdown/releaseonshutdown.h

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#include "../profiler.h"
+
+#include <unordered_set>
```

Comment:
nit: I don't think these headers ever used in this test?

Reply:
They aren't used, I think I used them in the first iteration of the test but it was before vacation and I can't remember why